### DISCUSSION TITLE
Cross references to html tables

### DISCUSSION THREAD

**Discussion by friosavila** [quarto-dev/quarto-cli#6610] [Q&A] [answered]
### Description

Im wondering if there is a way to cross reference to an HTML table. 

For example, with Markdown tables I would use the following

```
| Col1 | Col2 | Col3 |
|------|------|------|
| A    | B    | C    |
| E    | F    | G    |
| A    | G    | G    |

: My Caption {#tbl-letters}

See @tbl-letters.
```

and the last sentence would render the correct link to the table. 

The question is, if I have an HTML table:

```{=html}
<table>
...
</table>
```

where or how should I put the information equivalent to {#tbl-name} that would allow me to do references to it later in the text?

Thank you

**Comment by friosavila** [comment]
So im assuming there is none.
Thank you anyways. Perhaps that could be added as a feature in the future.

**Reply by cscheid** [reply, marked as answer]
(We're busy - sorry.) Keep an eye on #6620.

**Reply by friosavila** [reply]
Thank you! Will keep an eye on it!
